modules/document_repository/php/uploadcategory.class.inc

```diff
@@ -83,8 +83,8 @@ class UploadCategory extends \NDB_Page
         $DB            = \Database::singleton();
         $req           = $request->getParsedBody();
         $category_name = $req['categoryName']; // required
-        $parent_id     = $req['parentId']  !== '' ? $req['parentId'] : '0';
-        $comment       = $req['comments']    !== '' ? $req['comments'] : null;
+        $parent_id     = isset($req['parentId']) ? $req['parentId'] : '0';
+        $comments      = isset($req['comments']) ? $req['comments'] : null;
```

Comment:
are we keeping the same logic or intentionally changing it? isset doesn't check if a variable is empty. it only checks if it exists and is not null.  `$comment = $req['comments']  !== '' ? $req['comments'] : null` would be equivalent to `$comment = $req['comments'] ?? null`

Reply:
This is exactly what I need. React post form only passes the params which set values.